doc/source/examples/incompressible-flow/3d-flow-around-sphere/flow-around-sphere.rst

```diff
@@ -0,0 +1,329 @@
+
+.. code-block:: text
+
+  gls_navier_stokes_3d sphere_0.1.prm
+
+or 
+
+.. code-block:: text
+
+  gls_navier_stokes_3d sphere_150.prm
+
+or
+
+.. code-block:: text
+
+  gls_navier_stokes_3d sphere_adapt.prm
+
+Lethe will generate a number of files. The most important one bears the extension ``.pvd``. It can be read by popular visualization programs such as `Paraview <https://www.paraview.org/>`_. 
+
+
+First case results (Re = 0.1)
+-----------------------------
+
+Using Paraview, the steady-state velocity profile and the pressure profile can be visualized by operating a *slice* along the xy-plane (z-normal) that cuts in the middle of the sphere (See documentation <https://forgeanalytics.io/blog/creating-slices-in-paraview/>)_.
+
+.. image:: images/velocity_0.1.png
+    :alt: velocity distribution 0.1
+    :align: center
+
+.. image:: images/pressure_0.1.png
+    :alt: pressure distribution 0.1
+    :align: center
+
+We can appreciate the axisymmetrical behavior of the flow. The drag on the sphere is available in the output file ``force.00.dat`` (the other force files ``force.01.dat`` and ``force.02.dat`` give the forces on boundary conditions 1 and 2 respectively). 
+
+.. note::
+  The last line of the file shows the force calculated in the last iteration. Since the flow in the x-direction, the x-direction force ``f_x`` gives the drag force.
+
+.. code-block:: text
+
+  cells      f_x           f_y          f_z      
+  5823 98.3705224612 -0.0000000785 0.0000001119
+
+Given the flow parameters, the calculated drag coefficient is 250.50, using 6000 cells. At Re=0.1, an analytical solution of the drag coefficient is known: :math:`C_D = 240`. The deviation from the analytical solution is primarily due to the coarseness of the mesh. It would be relevant to carry out a mesh refinement analysis.
+
+Second case results (Re = 150)
+------------------------------
+
+We now consider the case at a Reynolds number of 150. At this value of the Reynolds number, the flow has separated, resulting in an unstable wake and recirculation. 
+
+The velocity and pressure are once again visualised as well as the mesh used:
+
+.. image:: images/velocity_150.png
+    :width: 93.8%
+    :alt: velocity distribution 150
+    :align: center
+.. image:: images/velocity_mesh_150.png
+    :alt: velocity distribution mesh 150
+    :align: center
+
+.. image:: images/pressure_150.png
+    :alt: pressure distribution 150
+    :align: center
+
+The drag coefficient at Re=150 using this example simulation is 0.798, against an analytical solution of 0.889.
+
+Third case results (Re = 150 with an adaptative mesh refinement)
+----------------------------------------------------------------
+
+Using mesh adaptative refinement, the final mesh contains slightly more than 101,000 cells. The resulting velocity profile is shown without and with the underlying mesh. Refinement around the sphere and wake can be observed:
```

Comment:
Normal that you have 101k elements if you limited the number to 50k ?

Reply:
Ah

Reply:
@blaisb

Reply:
it is set to 100,000, which is a bit below 101k. I modified the parameters.